README.md

```diff
@@ -113,8 +113,8 @@ as single result. This will work only for PDF files, so don't set it for other f
 Warning: this feature causes the `parent_id` metadata generation in elements to be disabled, as that
```

Comment:
I would also add information that split is done by default and can be disabled by setting `split_pdf_page` to `false`.

Reply:
done